_data/projects/constrained-database-with-slow-streams.yml

```diff
@@ -0,0 +1,8 @@
+name: Constrained Data Base with Slow Streams
+desc: Discussions (help-wanted) on usability of "Constrained Data Base with Slow Streams"
+site: https://github.com/bacloud14/Constrained-Data-Base-with-Slow-Streams
+tags:
+  - scientific
```

Comment:
@bacloud14 Can you please add more tags here. To make it more reachable.

Reply:
thanks for pointing out